metadeploy/api/models.py

```diff
@@ -210,17 +210,29 @@ def valid_token_for(self):
         return None
 
 
-class ProductCategory(models.Model):
-    title = models.CharField(max_length=256)
-    order_key = models.PositiveIntegerField(default=0)
-
+class ProductCategory(TranslatableModel):
     class Meta:
         verbose_name_plural = "product categories"
         ordering = ("order_key",)
 
+    order_key = models.PositiveIntegerField(default=0)
+    is_listed = models.BooleanField(default=True)
+
+    translations = TranslatedFields(
+        title=models.CharField(max_length=256),
+        description=MarkdownField(property_suffix="_markdown", blank=True),
```

Comment:
How about we mention that markdown is supported in the `help_text` for `description`?